plugins/comments.md

```diff
@@ -1,34 +1,34 @@
 ---
 layout: default
-title: Tiny Comments
+title: Comments
 title_nav: Comments
 description: Tiny Comments provides the ability to add comments to the content and collaborate with other users for content editing.
 keywords: comments commenting tinycomments
 ---
 
 ## Introduction
 
-The Tiny Comments plugin provides the user an ability to start or join a conversation by adding comments to the content within the TinyMCE editor. The Tiny Comments plugin is built upon the new [Annotations API]({{ site.baseurl }}/advanced/annotations/) and uses annotations to create comment threads (conversations).
+The Tiny Comments plugin provides the user an ability to start or join a conversation by adding comments to the content within the TinyMCE editor. The Comments plugin is built upon the new [Annotations API]({{ site.baseurl }}/advanced/annotations/) and uses annotations to create comment threads (conversations).
 
-This section describes the various configuration options for the Tiny Comments plugin.
+This section describes the various configuration options for the Comments plugin.
 
 ## Storage
 
-Like TinyMCE, the Tiny Comments plugin does not directly provide the user an ability to save the comments. You need to configure storage at your end to be able to save comments on your server. You can choose to configure your storage settings to either persist them immediately or save them at the same time as the content.
+Like TinyMCE, the Comments plugin does not directly provide the user an ability to save the comments. You need to configure storage at your end to be able to save comments on your server. You can choose to configure your storage settings to either persist them immediately or save them at the same time as the content.
 
-How you store those comments affects when other users see new comments. The Tiny Comments functions (create, reply, delete, and lookup) are configured differently depending upon the server-side storage configuration.
+How you store those comments affects when other users see new comments. The Comments functions (create, reply, delete, and lookup) are configured differently depending upon the server-side storage configuration.
 
-In this chapter, we have provided examples of both ways of configuring Tiny Comments storage.
+In this chapter, we have provided examples of both ways of configuring Comments storage.
 
 ### Storage - persist in real-time
 
-Here is a demo to showcase the Tiny Comments functionality using storage configured to persist in real-time:
+Here is a demo to showcase the Comments functionality using storage configured to persist in real-time:
 
 {% include codepen.html id="pOzxJw" %}
 
 ### Storage - persist on content-save
 
-Here is a demo to showcase the Tiny Comments functionality using storage configured to persist on content-save.
+Here is a demo to showcase the Comments functionality using storage configured to persist on content-save.
```

Comment:
"Here is a demo showcasing"? 
"Here is a demo which showcases"?
"The following demo showcases"?